docs/grisp_connect_api.md

```diff
@@ -71,3 +72,14 @@ Additionally to the default jsonrpc error codes the following codes will be retu
 | -2    | `"token expired"`         | token is expired                          |
 | -3    | `"device already linked"` | device needs to be unlinked first via UI  |
 | -4    | `"invalid token"`         | token is e.g. not orderly encoded         |
+
+## Notifications
+
+<details><summary><i>Notify - Status Update </i></summary>
+
+**`result`**:  `JSON Object`
+
+| key            | value             | description                       |
+| -------------- | ----------------- | --------------------------------- |
+| `"type"`       | `"status_update"` | Type of notification              |
+| `"percentage"` | integer           | Progress percentage of the update |
```

Comment:
This is a notification for the backend and is to be documented there, I forgot we already have one

Reply:
So I need to update this one too, right? https://github.com/stritzinger/grisp_manager/blob/main/docs/GRiSP-websocket-API.md